website/docs/policy/getting-started.mdx

```diff
@@ -45,7 +45,7 @@ You should then configure the `values.yaml`. You can find more about the policy
 ```
```

Comment:
Maybe another note here about how to install/enabled the policy agent in the mgmt cluster. (See the WGE installation instructions on how to enable policy).